descartes_samplers/include/descartes_samplers/evaluators/impl/distance_edge_evaluator.hpp

```diff
@@ -85,26 +58,17 @@ bool DistanceEdgeEvaluator<FloatType>::evaluate(const Rung_<FloatType>& from,
                    }
                  });
 
-  // Allocate
-  edges.resize(n_start);
-
-  for (std::size_t i = 0; i < n_start; ++i)
+  FloatType cost = static_cast<FloatType>(0.0);
+  for (std::size_t i = 0; i < delta_thresholds.size(); ++i)
   {
-    const auto* start_vertex = from.data.data() + dof * i;
-    for (std::size_t j = 0; j < n_end; ++j)
-    {
-      const auto* end_vertex = to.data.data() + dof * j;
+    FloatType step = end[i] - start[i];
+    if (std::abs(step) > delta_thresholds[i])
+      return std::make_pair(true, cost);
 
-      // Consider the edge:
-      considerEdge(start_vertex, end_vertex, delta_thresholds, j, edges[i]);
-    }
+    cost += std::pow(step, FloatType(2));
```

Comment:
What is the difference between this evaluator and the Euclidean distance evaluator? Just the fact that this one takes timing into account?

Reply:
Yes this just takes timing into account.

Reply:
Timing really should be a separate edge evaluator like you mentioned above. It's confusing that there are two evaluators with similar names that do the same thing except one takes timing into account

Reply:
Done.